nifi-extension-bundles/nifi-workday-bundle/nifi-workday-processors/pom.xml

```diff
@@ -78,6 +78,11 @@
             <artifactId>nifi-schema-registry-service-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-oauth2-provider-api</artifactId>
+            <scope>compile</scope>
```

Comment:
This scope should be changed to `provided` and declared before other test dependencies for consistency.